commands/azureCommands/pull-from-azure.ts

```diff
@@ -0,0 +1,38 @@
+import vscode = require('vscode');
+import { reporter } from '../../telemetry/telemetry';
+const teleCmdId: string = 'vscode-docker.image.pullFromAzure';
+import { AzureImageNode } from '../../explorer/models/AzureRegistryNodes';
+import * as acrTools from '../../utils/Azure/acrTools';
+
+/* Pulls an image from Azure. The context is the image node the user has right clicked on */
+export async function pullFromAzure(context?: AzureImageNode): Promise<any> {
+
```

Comment:
Is it possible to check if docker exists on the path at the beginning? I'm thinking prompt an error message if it doesn't. But I can also image the terminal below will also tell the docker doesn't exists. So what I suggest is compare the two user experience and see which make senses.

Reply:
Not going to include that now, since other commands like push image don't have it and it's heavy to include in this PR, as we discussed